examples/llama/train_llama2.sh

```diff
@@ -0,0 +1,338 @@
+#!/bin/bash
+###############################################################################
+#
+# MIT License
+#
+# Copyright (c) 2024 Advanced Micro Devices, Inc.
+#
+# Permission is hereby granted, free of charge, to any person obtaining a copy
+# of this software and associated documentation files (the "Software"), to deal
+# in the Software without restriction, including without limitation the rights
+# to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+# copies of the Software, and to permit persons to whom the Software is
+# furnished to do so, subject to the following conditions:
+#
+# The above copyright notice and this permission notice shall be included in all
+# copies or substantial portions of the Software.
+#
+# THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+# IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+# FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+# AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+# LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+# OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
+# SOFTWARE.
+#
+#################################################################################
+# set -x
+
+# set envs 
+export GPU_MAX_HW_QUEUES=2
+export TORCH_NCCL_HIGH_PRIORITY=1
+export NCCL_CHECKS_DISABLE=1
+export NCCL_IB_HCA=rdma0,rdma1,rdma2,rdma3,rdma4,rdma5,rdma6,rdma7 
+export NCCL_IB_GID_INDEX=3
+export NCCL_CROSS_NIC=0
+export NCCL_SOCKET_IFNAME=ens50f0np0 # network interface  
+export GLOO_SOCKET_IFNAME=ens50f0np0 # network interface
+export CUDA_DEVICE_MAX_CONNECTIONS=1
+export NCCL_PROTO=Simple
+export RCCL_MSCCL_ENABLE=0
+export TOKENIZERS_PARALLELISM=false
+export HSA_NO_SCRATCH_RECLAIM=1
+
+
+# parsing input arguments
+for ARGUMENT in "$@"
+do
+   KEY=$(echo $ARGUMENT | cut -f1 -d=)
+
+   KEY_LENGTH=${#KEY}
+   VALUE="${ARGUMENT:$KEY_LENGTH+1}"
+
+   export "$KEY"="$VALUE"
+done
+
+
+TIME_STAMP=$(date +"%Y-%m-%d_%H-%M-%S")
+EXP_NAME="${EXP_NAME:-perf}"
+
+TEE_OUTPUT="${TEE_OUTPUT:-1}"
+NO_TORCH_COMPILE="${NO_TORCH_COMPILE:-1}"
+USE_FLASH_ATTN="${USE_FLASH_ATTN:-1}"
+NO_TRAINING="${NO_TRAINING:-0}" # NO_TRAINING=1: for computing metrics only
+ENABLE_PROFILING="${ENABLE_PROFILING:-0}" #enable pytorch profiling
+echo "NO_TRAINING=$NO_TRAINING"
+
+CWD=`pwd`
+GPUS_PER_NODE=`python3 -c "import torch; print(torch.cuda.device_count())"`
+
+# single node config, Change for multinode config
+#MASTER_ADDR="${MASTER_ADDR:-localhost}"
+MASTER_ADDR="${MASTER_ADDR:-tw015}"
+MASTER_PORT="${MASTER_PORT:-6008}"
+#NNODES="${NNODES:-1}"
+NNODES="${NNODES:-2}"
```

Comment:
Looks like the default setting is multi-node setting. Could you please use single node setting as default for this one?